.appveyor.yml

```diff
@@ -57,14 +56,15 @@ install:
     7z x 3.3.3.zip -y > $null
     $env:CMAKE_INCLUDE_PATH = "eigen-eigen-67e894c6cd8f;$env:CMAKE_INCLUDE_PATH"
 build_script:
+- set VERBOSE=1
+- cmake --version
 - cmake -G "%CMAKE_GENERATOR%" -A "%CMAKE_ARCH%"
-    -DPYBIND11_CPP_STANDARD=/std:c++%CPP%
+    -DPYBIND11_CPP_STANDARD=/std:c++%CPP
```

Comment:
You stripped the trailing `%` here. `CPP` isn't treated as a variable in this case.

Reply:
Thanks. Committed too quickly!